src/apiCalls.js

```diff
@@ -1,4 +1,29 @@
 const BASE_URL = 'https://www.dnd5eapi.co';
+const LOCAL_BASE_URL = 'http://localhost:8080';
+
+const getStaffBios = () => {
+  return fetch(LOCAL_BASE_URL + '/api/staff')
+    .then((response) => {
+      if (!response.ok) {
+        if (response.status >= 400 && response.status < 500) {
+          throw new Error('Uh oh! Looks like you rolled a Natural 1. Please try again.');
```

Comment:
this is adorable and I love it, maybe you should include a button to refresh the page? no clue if youve already implemented this down the line

Reply:
@Jesuitman The app title on the header is a link to the home page. Do you think I need something else, too?

Reply:
@Jesuitman Side note, I'm glad you like my silly error messages. 😂

Reply:
Since your header is a link then i think that is perfect! no notes otherwise!